spec/support/liquid_shared_examples.rb

```diff
@@ -9,14 +9,15 @@
     visit @path
   end
 
-  it 'dynamically renders item properties' do
+  it 'dynamically renders item properties', js: true do
+    expect(page).to have_no_css('span.text-nowrap', text: 'Loading liquid dynamic content', wait: 10)
```

Comment:
10 seconds is only the maximum wait time and it won't really take this long. We're only setting the value here because [2 seconds](https://github.com/teamcapybara/capybara/blob/master/README.md#asynchronous-javascript-ajax-and-friends) might be too short and we need to account for possible hitches in CI that could cause a false negative.